liberty-ls/src/main/java/io/openliberty/tools/langserver/utils/ServerConfigUtil.java

```diff
@@ -9,24 +9,22 @@
 import javax.xml.xpath.XPathExpressionException;
 
 import org.w3c.dom.Document;
-import org.xml.sax.SAXException;
 
 import io.openliberty.tools.common.plugins.config.ServerConfigDocument;
 import io.openliberty.tools.langserver.CommonLogger;
 
 //TODO: rename class to relate to ci.common
 public class ServerConfigUtil {
-    
-    private static final Logger LOGGER = Logger.getLogger(ServerConfigUtil.class.getName());
-    static CommonLogger log = CommonLogger.getInstance(LOGGER);
+    private final static Logger LOGGER = Logger.getLogger(ServerConfigUtil.class.getName());
+    private static CommonLogger log = CommonLogger.getInstance(LOGGER);
+    private static ServerConfigDocument configDocument = new ServerConfigDocument(log);
```

Comment:
You're right. It's better design for each workspace to have an instance of `ServerConfigDocument` when we intend to use its properties down the line.

Reply:
I confused the existing concept of workspaces from Liberty LemminX to here. 
I've changed the routing so it's instanced each hover/completion. This can be improved upon later.

Reply:
So add a comment to the code that this MUST be changed to non-static before proceeding with the implementation.